App/styles.css

```diff
@@ -0,0 +1,73 @@
+body {
+    background-color: #330069;
+}
+
+#front-page {
+    margin-inline: auto;
+    display: flex;
+    flex-direction: column;
+    font-family: Verdana;
+    color: white;
+    width: 70%;
```

Comment:
U ovom slučaju (page container) ne bi koristili širinu u postotcima. Ona bi možda išla npr. ako imamo dva stupca di svaki zauzima 50% širine - primjer nekog ugovora, di je lijevo na hrvatskom, a desno na engleskom.

Možda bolje postaviti nešto kao:
```
width: 920px;
max-width: 100%;
```

Reply:
Riješeno, svugdje drugdje (gdje je potrebno) width i padding su u px, a ne u %